christfellowshipchurch/web-app-v2
# External Home Page 2.0

### About
This PR changes the UI for the new Value Buttons for the External Home Page.

### Test Instructions
1. Go to the External Home Page on [our live website](https://www.christfellowship.church/)
2. Go to the External Home Page on [the testing link](https://web-app-v2-git-home-value-heade-ce1a44-christ-fellowship-church.vercel.app//) and see the differences between the buttons.
3. Look at Mobile View as well.
4. Compare the Testing Link Buttons to Figma Design, they should look the same.

### Screenshots
|How it used to be|With the changes|
|--|--|
|<img width="1440" alt="Screenshot 2023-08-01 at 11 11 46 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/bcf51279-d32e-4182-ad9e-168e1a80ec43">|<img width="1440" alt="Screenshot 2023-08-01 at 11 12 54 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/a7365f9b-b961-4a6b-a744-e9891af43279">|

### Closes Tickets
[CFDP-2634](https://christfellowshipchurch.atlassian.net/browse/CFDP-2634)

[CFDP-2634]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2634?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
